Ds3/Helpers/Jobs/PartialReadJob.cs

```diff
@@ -30,7 +30,8 @@ internal class PartialReadJob : Job<IPartialReadJob, Ds3PartialObject>, IPartial
             JobResponse jobResponse,
             IEnumerable<string> fullObjects,
             IEnumerable<Ds3PartialObject> partialObjects,
-            ITransferItemSource transferItemSource)
+            ITransferItemSource transferItemSource,
+            int getObjectRetries = 5)
```

Comment:
Like Ds3ClientHelpers: publish the default as static DEFAULT_OBJECT_RETRIES = 5 ?